twilio/base/version.py

```diff
@@ -47,6 +47,23 @@ def request(self, method, uri, params=None, data=None, headers=None,
             allow_redirects=allow_redirects
         )
 
+    async def request_async(self, method, uri, params=None, data=None, headers=None,
+                            auth=None, timeout=None, allow_redirects=False):
+        """
+        Make an asynchronous HTTP request
+        """
+        url = self.relative_uri(uri)
+        return await self.domain.request_async(
```

Comment:
The `await` is needed here because we're calling a coroutine function that returns coroutine object (like a promise). The `await` operator schedules the coroutine to be executed, suspends the execution of the current surrounding coroutine (`Version.request_async` in this situation), and passes control back to the event loop. When the awaited coroutine finishes execution, control is passed back to the surrounding coroutine.